plugins/modules/proxmox.py

```diff
@@ -415,11 +424,23 @@ def is_template_container(self, node, vmid):
         return config['template']
 
     def create_instance(self, vmid, node, disk, storage, cpus, memory, swap, timeout, clone, **kwargs):
+        # Available only in PVE 7
+        only_v7 = ['tags']
```

Comment:
How about
```suggestion
        minimum_version = {
            'tags': 7,
        }
```
so that it is easy to add more checks also for different versions?